build-system/tasks/extension-generator/bento/index.js

```diff
@@ -125,6 +126,9 @@ async function makeBentoExtension() {
     log(green('SUCCESS:'), 'Created file', cyan(destination));
   }
 
+  insertExtensionBundlesConfig({name: `amp-${componentName}`, version});
+  log(green('SUCCESS:'), 'Wrote', cyan('bundles.config.js'));
```

Comment:
One change I forgot to request: Since it's done already, let's remove the section below describing the change to this file (lines 149-164 at the moment)